packages/client-sdk-nodejs/test/unit/cache-client.test.ts

```diff
@@ -61,4 +62,18 @@ describe('CacheClient', () => {
       }
     }
   });
+  it('createWithEagerConnections returns a client even if it cannot connect', async () => {
+    const momento = await CacheClient.createWithEagerConnections({
+      configuration: configuration,
+      credentialProvider: credentialProvider,
+      defaultTtlSeconds: 100,
+    });
+    const setResponse = await momento.set('cache', 'key', 'value');
+    expect(setResponse).toBeInstanceOf(CacheSet.Error);
```

Comment:
I did not totally understand why this will not succeed

Reply:
The token that this client uses is fake. It parses, so the client can be created, but it gets an error when it tries to connect.

Reply:
does it make sense for this to be a unit test then? Since it is attempting a network call? should it be an int test, and would it be worth adding a happy path int test or two?

These don't have to be addressed in this PR, can be a follow-up; for this PR the most important thing is that we align on the new API.